stubs/python-xlib/Xlib/threaded.pyi

```diff
@@ -0,0 +1,3 @@
+# We change the allocate_lock function in Xlib.support.lock to
+# return a basic Python lock, instead of the default dummy lock
```

Comment:
Not sure I understand this comment

Reply:
I copied the comment from source, but I'll modify it a bit to better explain why lock is re-exported in stubs (and that it's not a private import)